lib/beaker/host_prebuilt_steps.rb

```diff
@@ -338,7 +338,7 @@ def enable_root_login host, opts
         if host['platform'] =~ /debian|ubuntu|cumulus/
           host.exec(Command.new("sudo su -c \"service ssh restart\""), {:pty => true})
         elsif host['platform'] =~ /centos|el-|redhat|fedora|eos/
-          host.exec(Command.new("sudo -E /sbin/service sshd restart"))
+          host.exec(Command.new("sudo -E /sbin/service sshd restart"), {:pty => true})
```

Comment:
I think that this could be split out as a separate patch, away from the vagrant-openstack request.